### ISSUE TITLE
pretrained model on mxnet

### ISSUE THREAD

**Issue by shipeng-uestc** [CSAILVision/places365#36]
The pretrained model by mxnet ,resnet-50' accuracy is only 0.3112 and resnet-152' accuracy is only 0.3355.
the site is https://github.com/apache/incubator-mxnet/tree/master/example/image-classification.
So, I want use pretrained model by pytorch ,but ,how to convert pytorch's model to mxnet's model.
Thank you

**Comment by metalbubble** [comment]
If you look at the pretrained model on mxnet more CAREFULLY, you won't ask this kind of question. Thanks.

**Comment by shipeng-uestc** [comment]
Thank your answer.The pretrained model on mxnet has more images .It uses 20 million images,not 8 million
